Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -0,0 +1,100 @@
+## 195: Accessibility improvements in Windows Forms controls
+
+### Scope
+Major
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Windows Forms Framework is improving how it works with accessibility technologies to better support Windows Forms customers. Specifically, there are changes to improve display during High Contrast mode, to improve the property browser experience and to implement missing UI accessibility patterns in controls. Property Browser improvements will include better keyboard navigation though the various drop down selection windows, reduced unnecessary tab stops, better reporting of control types, and improved Narrator behavior. 
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+__How to opt in or out of these changes__
+  
+In order for the application to benefit from these changes, it should run on the 4.7.1 version of the Framework and be either re-compiled against the 4.7.1 .Net Framework, or application should opt out from the legacy, or 4.7 and below, accessibility behaviors by setting the following [AppContext Switch](https://msdn.microsoft.com/en-us/library/system.appcontext(v=vs.110).aspx) is a .Net compatibility mechanism.  to false. Add an app.confog file to your application and add the following switch to the '<runtime/>' section:
```

Comment:
Update link to docs.microsoft.com, like:
https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element

Reply:
done